Automattic/wp-calypso
# Update legacy plan redirect to include recommended items

### Changes proposed in this Pull Request

This PR adds the recommended items as a value of the `compare_plans` parameter, when redirecting users who hit the checkout page with a legacy plan to the plans page.

Fixes 1200196139286276-as-1200259578331173

### Testing instructions

- Download the PR and run Calypso
- Visit `/checkout/:site/:legacy-product`
- Check that you're redirected to the plans page, and see the upgrade section with the proper products (reference: https://jetpack.com/support/jetpack-plan-equivalency-chart/)
- Check the remaining legacy plans

### Screenshots


https://user-images.githubusercontent.com/1620183/116421483-dfa2e280-a80c-11eb-9611-e787bab10c64.mov
